src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -154,11 +154,11 @@ public string UniqueId
         /// <summary>
         /// Get the Element Pinned status
         /// </summary>
-        public bool IsPinned
+        public bool? IsPinned
```

Comment:
think we should not make this `nullable`, but default to `false` if the element is null